conda_smithy/ci_register.py

```diff
@@ -379,7 +379,7 @@ def add_conda_forge_webservice_hooks(user, repo):
             "http://conda-forge.herokuapp.com/conda-forge-command/hook",
             [
                 "pull_request_review", "pull_request",
-                "pull_request_review_comment", "issue_comment", "issue",
+                "pull_request_review_comment", "issue_comment", "issues",
```

Comment:
Honestly we might want to drop the [`issues` event]( https://developer.github.com/v3/activity/events/types/#issuesevent ). It seems more focused on metadata about the issue than anything that would be considered an actionable comment.

Reply:
There is `issue[title]` and `issue[body]` there. Things like rechecking the team can be commands given in an issue.

Reply:
So you are thinking of issue ( https://github.com/conda-forge/conda-forge-webservices/issues/104 ).

Reply:
Yes, exactly

Reply:
We should definitely discuss that use case more. That said, I'm ok with going ahead with this change. There may be some other things to explore like assignees and review requests.